instructor/templates/instructor/strain_protocols.html

```diff
@@ -1,19 +1,79 @@
 {% extends "instructor/assignment_sidebar.html" %}
 
 {% block right_side %}
-    You only get 5 questions so ask wisely.
-    {% if message %}
-        {{ messsage }}
-    {% endif %}
+
+
+
+    <div class='scb_s_assignment_setup_course_name_heading'>4. Below are all possible combinations of strains and
+        treatment protocols.
+    </div>
+    <div class='scb_s_experiment_setup_subtitle'>You can delete irrelevant combinations and the combination
```

Comment:
But the word is missing from the description, after `... irrelevant combinations and`